Sources/NIO/Selector.swift

```diff
@@ -41,6 +41,192 @@ private extension Optional {
     }
 }
 
+/// Represents IO events NIO might be interested in. `SelectorEventSet` is used for two purposed: To express interest
+/// in a given event set and for notifications about an IO event set that has occured.
+///
+/// For example, if you were interested in reading and writing data from/to a socket and also obviously if the socket
+/// receives a connection reset, express interest with `[.read, .write, .reset]`.
+/// If then suddenly the socket becomes both readable and writable, the eventing mechanism will tell you about that
+/// fact using `[.read, .write]`.
+struct SelectorEventSet: OptionSet, Equatable {
+
+    typealias RawValue = UInt8
+
+    let rawValue: UInt8
```

Comment:
nit: Should this be `let rawValue: RawValue` ?

Reply:
it's the same but changing it